scripts/lint-examples/package.json

```
@@ -11,12 +11,12 @@
     "@babel/core": "^7.25.2",
     "@babel/preset-env": "^7.25.4",
     "@babel/runtime": "^7.25.6",
-    "@react-native/babel-preset": "^0.76.0-rc.2",
     "@react-native-community/slider": "^4.5.3",
+    "@react-native/babel-preset": "^0.76.0-rc.2",
     "@react-native/eslint-config": "^0.76.0-rc.2",
     "@react-native/typescript-config": "^0.76.0-rc.2",
     "@types/react": "^18.2.79",
-    "eslint": "^8.57.1",
+    "eslint": "^8",
```

```suggestion
    "eslint": "^8.57.1",
```

Revert more granular range.